### DISCUSSION TITLE
Release v3.3.0

### DISCUSSION THREAD

**Discussion by thomas-bc** [nasa/fprime#2186] [Announcements]
## Breaking Changes
- fprime-arm-linux is now shipped in the core F´ framework. Users of the fprime-arm-linux external package should switch to using the toolchains shipped within F´.

## What's Changed
- New recommended CMake build structure, new projects are encouraged to use `fprime-util new --project`

## New features
- `fprime-util visualize`: visualize an F´ topology with a single command

## New Contributors
* @sobkulir made their first contribution in https://github.com/nasa/fprime/pull/1985
* @jwest115 made their first contribution in https://github.com/nasa/fprime/pull/1987
* @kubiak-jpl made their first contribution in https://github.com/nasa/fprime/pull/1944
* @NoahBraasch made their first contribution in https://github.com/nasa/fprime/pull/2024
* @ethancheez made their first contribution in https://github.com/nasa/fprime/pull/2070
* @hAkselS made their first contribution in https://github.com/nasa/fprime/pull/2095
* @mohitsingh999 made their first contribution in https://github.com/nasa/fprime/pull/2097
* @Boehm-Michael made their first contribution in https://github.com/nasa/fprime/pull/2098
* @dkogan made their first contribution in https://github.com/nasa/fprime/pull/2143
* @John-P-Cargille made their first contribution in https://github.com/nasa/fprime/pull/2147

**Full Changelog**: https://github.com/nasa/fprime/compare/v3.2.0...v3.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/nasa/fprime/releases/tag/v3.3.0'>Release v3.3.0</a>.</em>

**Comment by LeStarch** [comment]
🚀🚀🚀🚀🚀

**Comment by theonlychant** [comment]
let's go guys
